scheduler/src/cook/passport.clj

```diff
@@ -7,9 +7,13 @@
   "Log event to cook-passport log file"
   [log-data]
   (when (:enabled? (config/passport))
-    (log/log config/passport-logger-ns :info nil (json/write-str log-data))))
+    (log/log config/passport-logger-ns :info nil (json/write-str
+                                                   (assoc
+                                                     log-data
+                                                     :source :cook-scheduler
+                                                     :event-type (str "cook-scheduler/" (name (log-data :event-type))))))))
```

Comment:
Nitpick: you can destructure `:event-type` from the function argument

Reply:
I like that, will add.